pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -153,6 +153,13 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 			Tags: tags,
 		}
 
+		// Only set if specified so we don't change existing logic
+		if lbSpec.CrossZoneLoadBalancing != nil {
```

Comment:
**Small test:**
- Deploy with `crossZoneLoadBalancing` unset; by default the value on the ELB is set to false
- Set the above in kops spec with value `true`; this is successfully set to true in the ELB Spec
- Revert to original configuration (not explicitly setting in the LoadBalancerSpec to false); no change to the ELB configuration (the value remains true on the ELB resource)

Just a small suggestion that if `crossZoneLoadBalancing` is unset in the LoadBalancerSpec, the ELB property is explicitly set to false (maintains existing behaviour), so the above potential use-case is satisfied.

Reply:
Yeah you're correct. I'll make that change